internal/image/ostree_simplified_installer.go

```diff
@@ -0,0 +1,151 @@
+package image
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/osbuild/osbuild-composer/internal/artifact"
+	"github.com/osbuild/osbuild-composer/internal/common"
+	"github.com/osbuild/osbuild-composer/internal/disk"
+	"github.com/osbuild/osbuild-composer/internal/environment"
+	"github.com/osbuild/osbuild-composer/internal/fdo"
+	"github.com/osbuild/osbuild-composer/internal/manifest"
+	"github.com/osbuild/osbuild-composer/internal/platform"
+	"github.com/osbuild/osbuild-composer/internal/rpmmd"
+	"github.com/osbuild/osbuild-composer/internal/runner"
+	"github.com/osbuild/osbuild-composer/internal/workload"
+)
+
+type OSTreeSimplifiedInstaller struct {
```

Comment:
Potentially controversial, should this be `CoreOSInstaller`? It doesn't necessarily install CoreOS, but it does use the CoreOS installer.

Reply:
@nullr0ute wanted to rename it to "Simplified Provisioner", IIRC.

Reply:
Any name works for me.

Reply:
I think we should still make the specific image type "RHEL for Edge Simplified Provisioner" (or whatever it ends up being), but this is the internal image kind which should be generic (whether the payload is R4E or CoreOS or anything else).

Reply:
@teg but `CoreOSInstaller` is a bit ambiguous, because that exists as its own thing too; I mean it is used in a very specific way here that is different way here than in FOCS/RHCOS.

Reply:
Yup, it is ambiguous. I meant to refer to the name of the software, not the name of the artifact.
Think "Anaconda" rather than "Fedora Workstation Installer", in this case the terms are just a but overloaded.

Reply:
*scratches head* you might be right... let's table this for now anyway, not crucial to the PR at all :)